docs/reports/index.md

```diff
@@ -64,6 +64,10 @@ This page includes only experiments that have at least one run or report.
 - Hybrid Norm and Input Embedding Norm [![#961](https://img.shields.io/github/issues/detail/state/marin-community/marin/961)](https://github.com/marin-community/marin/issues/961)
   - [GitHub Issue #961](https://github.com/marin-community/marin/issues/961)
   - [WandB Report](https://wandb.ai/marin-community/hybrid-norm/reports/Hybrid-Norm--VmlldzoxMjY2MDgxMA)
+- OLMoE replication - MoE vs dense [![#1183](https://img.shields.io/github/issues/detail/state/marin-community/marin/1183)](https://github.com/marin-community/marin/issues/1183)
+  - [GitHub Issue #1183](https://github.com/marin-community/marin/issues/1183)
+  - [WandB Report](https://api.wandb.ai/links/marin-community/qi3u8nx7)
+  - Conclusion: MoE is slower but outperforms similar sized dense model in both training and evaluation.
```

Comment:
Unclear what slower means... you mean MFU...

Reply:
updated.